packages/ui/src/services/color-config/color-functions.ts

```diff
@@ -6,14 +6,14 @@ export const colorToRgba = (color: ColorInput, opacity: number) => {
   return new ColorTranslator(color).setA(opacity).RGBA
 }
 
-export const isLightBackground = (color: ColorInput) => {
+export const isLightBackground = (color: ColorInput, threshold: number) => {
   // TODO: replace with color mixin (doesn't play any role in dark theme)
   const { R, G, B } = new ColorTranslator(color)
-  return Math.sqrt(R * R * 0.241 + G * G * 0.691 + B * B * 0.068) > 120
+  return Math.sqrt(R * R * 0.241 + G * G * 0.691 + B * B * 0.068) > threshold
 }
 
-export const getTextColor = (color: ColorInput, darkColor = 'textDark', lightColor = 'textLight') => {
-  return isLightBackground(color) ? darkColor : lightColor
+export const getTextColor = (color: ColorInput, darkColor: string, lightColor: string, threshold: number) => {
+  return isLightBackground(color, threshold) ? darkColor : lightColor
```

Comment:
maybe we can set default values to `darkColor = 'textDark'`, `lightColor = 'textLight'`.
Shouldn't we use `threshold` from `globalConfig.colors.threshold` by default?

Reply:
It is used in useColors, cuz it we can use globalConfig only in setup context.